doc/api.md

```diff
@@ -15,6 +19,7 @@ To do so, please send an email specifying:
 * a username
 * an email address (in order to tell you if something wrong has been done with your key)
 * a short description of your project
+* if you need more than 30 requests/second rate
```

Comment:
Just out of curiosity, what do we do in this case? :)

Reply:
at this moment : add the IP adresse in the allowlist to avoid automatic ban
in the futur : Nginx limit management on IP list to avoid 30 rqs/s limit